conan/cli/formatters/graph/graph.py

```diff
@@ -104,7 +106,15 @@ def format_graph_html(result):
     template_folder = os.path.join(conan_api.cache_folder, "templates")
     user_template = os.path.join(template_folder, "graph.html")
     template = load(user_template) if os.path.isfile(user_template) else graph_info_html
-    cli_out_write(_render_graph(graph, template, template_folder))
+    error = None
+    if isinstance(graph.error, GraphConflictError):
+        error = {
+            "label": graph.error.require.ref,
+            "type": "conflict",
+            "from": graph.error.node.ref or graph.error.base_previous.ref,
```

Comment:
Not sure that providing the ``ref`` as ``from`` is the best approach. This can be problematic for graphs where there are multiple nodes with same ``ref`` which is totally possible. It seems that the ``node`` actual instance, or unique ``id`` should be used.

Reply:
I did it this way becuase I thought that the error id would not be valid, but alas! Works nicely after testing!